### PR TITLE
test: fix flaky window post tests

### PR DESCRIPTION
**Pull request by Stebalien**
Wait until the network upgrade is finished. If we try to seal while it's happening, we have a few annoying edge cases that can fail if we try to submit some messages right on the upgrade epoch (which is why everyone turns that kind of stuff off for the upgrade epoch).

### CODE REVIEW

**Review by Kubuxu — APPROVED** [review]

File: itests/wdpost_test.go [hunk 1]
```diff
@@ -268,6 +274,12 @@ func TestWindowPostBaseFeeBurn(t *testing.T) {
 	client, miner, ens := kit.EnsembleMinimal(t, kit.MockProofs(), opts)
 	ens.InterconnectAll().BeginMining(blocktime)
 
+	// Ideally we'd be a bit more precise here, but getting the information we need from the
+	// test framework is more work than it's worth.
+	//
+	// We just need to wait till all upgrades are done.
+	client.WaitTillChain(ctx, kit.HeightAtLeast(20))
```
Comment (raulk) [inline itests/wdpost_test.go:281]: @Stebalien we could add a predicate that tests against `StateNetworkVersion`? We'd need to expand the signature of `ChainPredicate` so we can pass in the FullNodeAPI.

Comment (Stebalien) [reply itests/wdpost_test.go:281]: Yeah. That would work. I guess I could also just pass a closure here and capture it.

Comment (raulk) [reply itests/wdpost_test.go:281]: Good idea.
